### PR TITLE
Change README.md to modify armv7l support and add aarch64 support

### PR DESCRIPTION
**Pull request by jam7**
Modify README.md as described in the title.  :)

### CODE REVIEW

**Comment by uberhacker** [comment]
Does this corrupt the spacing of the box?

**Comment by jam7** [comment]
No.  Browser is adjusting the spacing.

File: README.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ Supported Systems
 |:--------------:|:----------:|
 |     x86_64     |     Yes    |
 |      i686      |     Yes    |
-| arm (Exynos 5) |     Yes    |
+|     armv7l     |     Yes    |
+|     aarch64    |     Yes(use armv7l binaries)    |
```
Comment (flyingP0tat0) [inline README.md]: I think a whitespace after `Yes` should make it more readable.

Comment (flyingP0tat0) [inline README.md]: Second: although it does not break the table, I guess it is considered good practise to fix the spacing in the Markdown.

**Comment by uberhacker** [comment]
Do we even need the box?  Wouldn't a bulleted list work also?  The yes column is redundant anyway.

File: README.md (hunk 1 above)
Comment (jam7) [inline README.md]: Thanks.  I modified it.

**Comment by jam7** [comment]
@uberhacker Maybe yes, but I don't care it.  Please modify it once this PR is merged.

**Comment by RMcNeely** [comment]
Does this mean that Chromebrew is expected to work on ARM Chromebooks even without Exynos chipsets?

**Comment by jam7** [comment]
Originally, I ported chromebrew to c100pa which is RK3288 regarding to https://www.chromium.org/chromium-os/developer-information-for-chrome-os-devices.  Is it Exynos chipset?

**Pull request merged**